### DISCUSSION TITLE
Could not find a declaration file for module 'laravel-vue-pagination'

### DISCUSSION THREAD

**Discussion by DavidCCoimbra** [gilbitron/laravel-vue-pagination#156] [Help & support]
Is there any way to fix this?
I'm using Vue3 - Laravel, i don't know anything about TS. :(

![imagem](https://user-images.githubusercontent.com/88290682/205237628-ee09f5a3-3c10-4460-bf86-f1dd6d728d98.png)
![imagem](https://user-images.githubusercontent.com/88290682/205237764-c6f78b44-12df-4b3f-931c-9f312634d2cb.png)

**Comment by junedc** [comment]
@DavidCCoimbra , did you manage to fix this issue?  having the same error.  Thanks

**Reply by rendrah** [reply]
have you fix this issue.. i have same error

**Comment by marciomdrs2002** [comment]
Did anyone get it??
